Buform.Example.Core/ComponentsViewModel.cs

```diff
@@ -298,6 +299,13 @@ public ComponentsViewModel(ILoggerFactory logFactory, IMvxNavigationService navi
                         Label = "Time",
                         InputType = DateTimeInputType.Time
                     }
+                },
+                new TextFormGroup("Digit generation")
```

Comment:
I would move it to the menu form, as it demonstrates the usage of a custom header. And the components form is for embedded controls only.